tests/oauth2/rfc6749/endpoints/test_credentials_preservation.py

```diff
@@ -42,18 +42,6 @@ def set_client(self, request):
 
     def test_state_preservation(self):
         auth_uri = 'http://example.com/path?state=xyz&client_id=abc&response_type='
-        token_uri = 'http://example.com/path'
-
-        # authorization grant
-        h, _, s = self.web.create_authorization_response(
-                auth_uri + 'code', scopes=['random'])
-        self.assertEqual(s, 302)
-        self.assertIn('Location', h)
-        code = get_query_credentials(h['Location'])['code'][0]
```

Comment:
This (the authorization response) redirect, *should* contain a `state` as a query parameter if the authorization request included one. But I guess that is tested elsewhere already?

Reply:
No I had a bit overlooked and I removed too many lines :-) Let me fix thiw test there.